examples/Microsoft.CognitiveServices_accounts_raiPolicies@2024-10-01/main.tf

```diff
@@ -0,0 +1,76 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+  }
+}
+
+provider "azapi" {
+  skip_provider_registration = false
+}
+
+variable "resource_name" {
+  type    = string
+  default = "acctest0003"
+}
+
+variable "location" {
+  type    = string
+  default = "eastus"
+}
+
+resource "azapi_resource" "resourceGroup" {
+  type     = "Microsoft.Resources/resourceGroups@2020-06-01"
+  name     = var.resource_name
+  location = var.location
+}
+
+resource "azapi_resource" "account" {
+  type      = "Microsoft.CognitiveServices/accounts@2022-10-01"
+  parent_id = azapi_resource.resourceGroup.id
+  name      = var.resource_name
+  location  = var.location
+
+  body = {
+    identity = {
```

Comment:
Please remove the `identity` block inside the `body`

Reply:
Updated